### ISSUE TITLE
Blank image and low quality style models 

### ISSUE THREAD

**Issue by engahmed1190** [DmitryUlyanov/texture_nets#85]
Hi,
 i am trying to train models using Johnson model , bu what ever i try low images are appearing 
- i have tried to make the image width multiple of 32
- i have tried to decrease learning rate 

what is the best solution to get good results in my case

`th train.lua -data dataset -style_image ./AdobeStock_90768625.jpeg -style_size 600 -image_size 512 -model johnson -batch_size 4 -learning_rate 1e-2 -style_weight 10 -style_layers relu1_2,relu2_2,relu3_2,relu4_2 -content_layers relu4_2 -gpu 0
`
 
![Uploading ballon.jpg…](image used for styling)

**Comment by suke27** [comment]
what is low image
